src/commonMain/kotlin/operations/numeric/unwrap/LenientUnwrapStrategy.kt

```diff
@@ -0,0 +1,25 @@
+package operations.numeric.unwrap
+
+import asList
+
+internal interface LenientUnwrapStrategy: UnwrapStrategy {
+    override fun unwrapValues(wrappedValue: Any?): List<Any?> = wrappedValue.asList.map(::unwrap)
```

Comment:
hmmm I thought you'll mark somehow that the operator that uses it needs casting to number (so unwrapValues will return List<Double> or maybe List<Number>). I'm not sure how returning `List<Any?>` makes this code better but maybe it'll be more clear when you add casting to other types 🤔

Reply:
Alright, I added the casting version of the overridden function